### DISCUSSION TITLE
The $:/SiteTitle file contains special characters, causing login authentication failure

### DISCUSSION THREAD

**Discussion by oeyoews** [TiddlyWiki/TiddlyWiki5#8337] [Q&A] [answered]
https://github.com/Jermolene/TiddlyWiki5/blob/8928b6e603751001ddb0b9a4bfc76107d725dd95/core/modules/server/server.js#L96

This line of code seems to cause login authentication errors in special circumstances
![image](https://github.com/Jermolene/TiddlyWiki5/assets/72405338/2ad57e61-88ce-499f-8013-1a013e6976de)
I think this is caused by the html comments in the file.

![image](https://github.com/Jermolene/TiddlyWiki5/assets/72405338/4db7b133-abc1-4b53-9d00-a1482ce3e18b)

@Jermolene

**Comment by Jermolene** [comment]
Hi @oeyoews that is very curious. I would have expected `$tw.utils.transliterateToSafeASCII()` to prevent this from happening.

Please could you post the text of the `$:/SiteTitle` tiddler?

Until we have a fix you should be able to work around this problem by specifying an alternate server name on the command line with `server-name=TiddlyWikiStarterKit`

**Reply by oeyoews** [reply]
```html
<!-- <% if [{$:/info/url/protocol}!match[https:]] %>
🔴
<% endif %> -->
TiddlyWiki Starter Kit
```

**Comment by Jermolene** [comment, marked as answer]
Thanks @oeyoews I've created a ticket #8338
